crates/orchestrator/src/jobs/mod.rs

```diff
@@ -406,6 +408,15 @@ fn get_u64_from_metadata(metadata: &HashMap<String, String>, key: &str) -> color
         .wrap_err(format!("Failed to parse u64 from metadata key '{}'", key))
 }
 
+fn parse_string<T: FromStr>(value: String) -> Result<T, JobError>
```

Comment:
should we move this to the utils crate?

Reply:
Was not able to move to utils crate to cyclic dependecy as it needs to import `JobError` from `orchestrator` crate. Moved to conversion.rs